jdi-light-mobile-examples/src/main/java/selenium/site/sections/ContactForm.java

```diff
@@ -0,0 +1,142 @@
+package selenium.site.sections;
+
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import org.openqa.selenium.WebElement;
+import selenium.site.data.ContactInfo;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class ContactForm {
+    @UI("#name")
+    WebElement name;
+    @UI("#last-name")
+    WebElement lastName;
+    @UI("#position")
+    WebElement position;
+    @UI("#passport-number")
+    WebElement passportNumber;
+    @UI("#passport-seria")
+    WebElement passportSeria;
+    @UI("#passport")
+    WebElement passport;
+    // Dropdown
+    @UI("#gender")
+    WebElement genderExpand;
+    @UI("#gender") List<WebElement> genderList;
+    @UI("#gender")
+    WebElement genderValue;
+    // Combobox
+    @UI("#religion")
+    WebElement religion;
+    // MultiDropdown
+    @UI("#gender")
+    WebElement weatherExpand;
+    @UI("#gender") List<WebElement> weatherList;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)